tools/buildtools/ambertools/macros.xml

```diff
@@ -0,0 +1,25 @@
+<macros>
+    <token name="@VERSION@">0.19</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="19.0">ambertools</requirement>
+            <requirement type="package" version="3.0.0">libgfortran</requirement>
```

Comment:
Is this line needed? Tests are passing locally without it.

Reply:
Wasn't for me initially. Possibly good since you've put ambertools on conda-forge. will check

Reply:
Agreed libgfortran not a requirement.